lib/virtual_regional_office/configuration.rb

```diff
@@ -24,7 +24,6 @@ def connection
         # faraday.request(:curl, ::Logger.new($stdout), :warn) unless Rails.env.production?
         # faraday.response(:logger, ::Logger.new($stdout), bodies: true) unless Rails.env.production?
 
-        faraday.response :json
```

Comment:
This has been replaced by more subtle JSON behavior in `VirtualRegionalOffice::Client#perform` -- see the big code comment above that method.